### PR TITLE
Let WP make images responsive

### PR DESCRIPTION
**Pull request by ellatrix**
## Description

Let WP make images responsive. WP looks at the class with ID for this.

## How Has This Been Tested?
Upload an image, and view on the front-end. Verify it has `srcset` attributes etc.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style.
- [x] My code has proper inline documentation.

### CODE REVIEW

File: blocks/library/image/index.js [hunk 1]
```diff
@@ -162,9 +162,17 @@ export const settings = {
...
-		const { url, alt, caption, align, href, width, height } = attributes;
-		const extraImageProps = width || height ? { width, height } : {};
-		const image = <img src={ url } alt={ alt } { ...extraImageProps } />;
+		const { url, alt, caption, align, href, width, height, id } = attributes;
+
+		const image = (
+			<img
+				src={ url }
+				alt={ alt }
+				className={ id ? `wp-image-${ id }` : null }
```
Comment (mtias) [inline blocks/library/image/index.js]: wouldn't this invalidate existing image blocks?

Comment (iseulde) [reply blocks/library/image/index.js]: Yes, somehow didn't expect missing attributes to error.

**Comment by youknowriad** [comment]
closes #4163 and #869

**Review by youknowriad — APPROVED** [review]
LGTM 👍

**Comment by Adam-Collier** [comment]
Hey, does adding this class presume it is full width? Say you had something half the width of the viewport can we account for that too? My images seem bigger than they should be

**Pull request merged**